### DISCUSSION TITLE
Make trajectory format easier to parse

### DISCUSSION THREAD

**Discussion by jhenin** [Colvars/colvars#758] [Ideas]
Two ideas :
- use tabs to separate fields
- remove commas from vector values
  - surround vectors with {} instead of () for transparent Tcl parsing?

**Comment by giacomofiorin** [comment]
Binary format!

**Reply by jhenin** [reply]
Yes! Then we'll need to distribute a convenient parser/viewer for human inspection (say one in colvartools, one in VMD).
